js/utils/constants.js

```diff
@@ -1101,6 +1101,39 @@
       PLAYLISTS_READY: 'playlistReady',
 
       /**
+       * An attempt has been made to set session id. This will remain same until the player is destroyed.
+       *   <ul>
+       *     <li>The ID (session id) of the player.</li>
+       *   </ul>
+       *
+       *
+       * @event OO.EVENTS#SESSION_ID_SET
+       */
+      SESSION_ID_SET: "sessionIdSet",
+
+      /**
+       * An attempt has been made to set content session id. This will get sent out when new asset is loaded.
```

Comment:
- The comments in these three messages seem to all follow: "An attempt has been made to set..."
We have `SESSION_ID_SET` above; should we be consistent here and also have `CONTENT_SESSION_ID_SET` and `AD_BREAK_SESSION_ID_SET`. 

- As a follow-up question. The comment suggests that an "attempt" was made, but the message itself suggests that it has been already been set.